improver_tests/acceptance/SHA256SUMS

```diff
@@ -164,15 +164,15 @@ fe84db49d69ea62e2e82312c7b3d860c7329c627d79d76c2543d365f69d2e0d8  ./estimate-emo
 fc0d25a60a0ab753d019310c0e718110a6d994b2e8e6147e6262a0e6eea794e8  ./extract/basic/kgo.nc
 925c95be9546a6e3034fa50a80b61a5330fa1e6e1ecd24ae31e0adca5eb0cddf  ./extract/change_units/kgo.nc
 9db96e63e4235d410941deadc16ed9608df91f33ddf7656acd9fa3971e9ff3bc  ./extract/grids/input_grid_latlon.nc
-073bec02083dfaacf432d544a9ea9faace10a0c614f3425be408d22310789570  ./extract/grids/input_grid_uk.nc
+6af6442eba0ecc2079c7ad850606254cb5ce15f0b584a9d93821b098d2c6081f  ./extract/grids/input_grid_uk.nc
```

Comment:
The `Conventions` attribute has been updated. In total, this attribute has been updated in:

- extract/grids/kgo_grid_uk.nc
- extract/grids/input_grid_uk.nc
- extract/sites/kgo_spot.nc
- extract/sites/input_spot.nc
- weighted_blending/percentile_weights_from_dict/kgo.nc
- weighted_blending/spatial_weights/kgo/model.nc
- weighted_blending/spatial_weights/kgo/model_no_fuzzy.nc
- weighted_blending/three_models/kgo.nc
- weighted_blending/weights_from_dict/kgo.nc
- weighted_blending/model/kgo.nc

I think that this is fine.

Reply:
FYI there is more information about this on the ticket.